utils/quick_setup.sh

```diff
@@ -61,11 +61,9 @@ if test -f "$VPP"; then
   AIETOOLS="`dirname $VPP`/../aietools"
   mkdir -p my_install
   pushd my_install
-  # pip download mlir_aie -f https://github.com/Xilinx/mlir-aie/releases/expanded_assets/latest-wheels/
-  wget -q --show-progress https://github.com/Xilinx/mlir-aie/releases/download/latest-wheels/mlir_aie-0.0.1.2024061822+29d5cec-py3-none-manylinux_2_35_x86_64.whl
```

Comment:
could we get to the point where we just pip install the packages into the virtual environment, rather than this dance?

Reply:
I just looked into this and if I do
```
  pip install mlir_aie -f https://github.com/Xilinx/mlir-aie/releases/expanded_assets/latest-wheels/
```
it actually picks up the right mlir-aie version and installs it. However, it does not seem to add `aiecc.py` where it does currently and so it's not found on the PATH.

Given that, is it cool if I leave this as-is for now?